extension/src/popup/components/account/AccountHeader/index.tsx

```diff
@@ -8,13 +8,23 @@ import { useTranslation } from "react-i18next";
 
 import { ROUTES } from "popup/constants/routes";
 import { LoadingBackground } from "popup/basics/LoadingBackground";
+import { isActiveNetwork } from "helpers/stellar";
 import { AccountListIdenticon } from "popup/components/identicons/AccountListIdenticon";
-import { settingsNetworkDetailsSelector } from "popup/ducks/settings";
+import {
+  changeNetwork,
+  settingsNetworkDetailsSelector,
+  settingsNetworksListSelector,
+} from "popup/ducks/settings";
 import { AccountList } from "popup/components/account/AccountList";
+import { AccountHeaderModal } from "popup/components/account/AccountHeaderModal";
 import IconCube from "popup/assets/icon-cube.svg";
 
 import "./styles.scss";
 
+const NETWORK_COLORS = ["#30A46C", "#6E56CF"];
+
+const getNetworkColor = (index: number) => NETWORK_COLORS[index];
```

Comment:
this is WIP. In the next PR, these method will be used to generate a colored dot next to the network name